src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -1359,6 +1360,8 @@ def aks_browse(cmd, client, resource_group_name, name, disable_browser=False, li
         subprocess.call(["kubectl", "--kubeconfig", browse_path, "--namespace", "kube-system",
                          "port-forward", dashboard_pod, "{0}:9090".format(listen_port)])
     except KeyboardInterrupt:
+        # TODO: Better error handling here.
+        requests.post('http://localhost:8888/closeport/8001')
```

Comment:
Shouldn't this be inside an `if in_cloud_console()` block, and maybe in a `finally:`? This is actually the "normal" exit path since users are expected to hit [Ctrl+C] to terminate the proxy process.

```python
except KeyboardInterrupt:
    # Let command processing finish gracefully after the user presses [Ctrl+C]
    return
finally:
    if in_cloud_console():
        # TODO: Better error handling here.
        requests.post('http://localhost:8888/closeport/8001')
```